web_tree_image_tooltip/i18n/web_tree_image.pot

```diff
@@ -0,0 +1,14 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 11.0\n"
```

Comment:
Odoo Server 12.0

Reply:
@sudhir-erpharbor I think here @OCA-git-bot change automatically.

Reply:
Yes transifex would do it.

Reply:
Sounds good.

Reply:
Well, for being precise, it will be changed by Weblate when the first change in the translations is done, and it will be changed by a Weblate text, not Odoo version. Take into account that this text is ignored by default and it's only put as reference.